docs/resources/metal_device_network_type.md

```diff
@@ -13,27 +13,26 @@ To learn more about Layer 2 networking in Equinix Metal, refer to
 
 If you are attaching VLAN to a device (i.e. using equinix_metal_port_vlan_attachment), link the device ID from this resource, in order to make the port attachment implicitly dependent on the state of the network type. If you link the device ID from the equinix_metal_device resource, Terraform will not wait for the network type change. See examples in [equinix_metal_port_vlan_attachment](port_vlan_attachment).
```

Comment:
Was hoping that the addition of `DeprecationMessage` would feed into generated docs, but alas.  Should keep the hand-written template for now and delete it all when the resource is eventually removed.